src/actions/callsuborchestratoraction.ts

```diff
@@ -0,0 +1,11 @@
+import { ActionType, IAction } from "../classes";
+
+export class CallSubOrchestratorAction implements IAction {
+    public readonly actionType: ActionType = ActionType.CallSubOrchestrator;
+
+    constructor(
+        public readonly functionName: string,
+        public readonly instanceId: string,
+        public readonly input?: unknown,
+    ) { }
```

Comment:
Is this a public type to customers? If so, you should make sure you have a unit test to make sure you don't break the public surface area.

Reply:
Apply this same question to all the these dto type classes

Reply:
This is not a public type to customers; they should never interact with it directly.